frontend/src/components/pages/acls/Acl.List.tsx

```diff
@@ -593,24 +605,88 @@ const RolesTab = observer(() => {
   );
 });
 
-const AclsTab = observer((p: { principalGroups: AclPrincipalGroup[] }) => {
-  useEffect(() => {
-    void api.refreshAcls(AclRequestDefault, true);
-  }, []);
+interface SimpleAcl {
+  host: string;
+  principal: string;
+  principalType: string;
+  principalName: string;
+  hasAcl: boolean;
+}
+
+const AclsTab = observer((_: { principalGroups: AclPrincipalGroup[] }) => {
```

Comment:
I would strongly advise to reconsider if we want to continue adding tech debt with the `observer()` MobX pattern. I think this is not going to be easy to maintain

Reply:
Ok, maybe we can only make that specific part wrapped with an `observer()` MobX pattern